easybuild/easyconfigs/x/xtb/xtb-6.2.3-foss-2019b.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
```

Please, change name capitalization here and in the file name to match official name. 
```suggestion
name = 'xTB'
```

---

Please, the name of the program is still `xtb`.

---

No problem, the name can stay as `xtb`. We already have older easyconfigs for it with that name as well.